src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py

```diff
@@ -0,0 +1,170 @@
+#
```

Comment:
running this unit test returns 32 warnings invalid values:

```
(.venv) lambertSurfaceRelativeVelocity/_UnitTest % pytest
============================= test session starts ==============================
platform darwin -- Python 3.11.9, pytest-8.3.2, pluggy-1.5.0
rootdir: /Users/hp/Repos/basilisk/src
configfile: pytest.ini
plugins: html-4.1.1, metadata-3.1.1, xdist-3.6.1
collected 32 items                                                             

test_lambertSurfaceRelativeVelocity.py ................................  [100%]

=============================== warnings summary ===============================
fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py: 16 warnings
  /Users/hp/Repos/basilisk/src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py:85: RuntimeWarning: invalid value encountered in divide
    omegaHat_PN_N = omega_PN_N/np.linalg.norm(omega_PN_N)

fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py: 16 warnings
  /Users/hp/Repos/basilisk/src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py:136: RuntimeWarning: invalid value encountered in divide
    s1Hat = np.cross(omega_PN_N, r0_BN_N)/np.linalg.norm(np.cross(omega_PN_N, r0_BN_N))

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```

Reply:
Good catch. The reason for this was that the surface frame is ill-defined if the angular velocity vector of the celestial body is zero. I fixed this by using the inertial z vector [0, 0, 1] for the frame definition if the angular velocity is close to zero.